monitoring/src/check-page/tcfv2.ts

```diff
@@ -133,12 +137,15 @@ const checkPages = async (config: Config, url: string, nextUrl: string) => {
 	await checkCMPLoadingTime(page, config);
 
 	await page.close();
-
+} catch (e) {
```

Comment:
I think the formatting here is a little funky, I don't think this project has a "format" task to run prettier.